lib/prometheus/business_collector.rb

```diff
@@ -84,6 +101,50 @@ def collect
     @client_policies.observe(
       Policy.where(account_id: client_accounts.select(:id)).count
     )
+
+    sq = Policy.joins(:profiles).select(:account_id, "profiles.ref_id").distinct
+    Policy.select('sq.ref_id', 'COUNT(sq.account_id)')
+          .from("(#{sq.to_sql}) as sq")
+          .group('sq.ref_id').each do |item|
+      @total_policies_by_account.observe(
+        item.attributes['count'], ref_id: item.attributes['ref_id']
+      )
+    end
+
+    sq = Policy.where(account_id: client_accounts.pluck(:id))
```

Comment:
Observation: Variable override.
I guess we should start to spit these to separate methods :slightly_smiling_face:.

Reply:
I created an issue for that yesterday :wink: